receiver/elasticsearchreceiver/testdata/expected_metrics/noNodes.json

```diff
@@ -192,6 +192,246 @@
+                              "timeUnixNano": "1661811689943245000"
+                           }
+                        ]
+                     },
+                     "unit": "{operations}"
+                  },
+                  {
+                     "description": "Time spent on operations for an index.",
+                     "name": "elasticsearch.index.operations.time",
+                     "sum": {
+                        "aggregationTemporality": "AGGREGATION_TEMPORALITY_CUMULATIVE",
+                        "isMonotonic": true,
+                        "dataPoints": [
+                           {
+                              "asInt": "82",
+                              "attributes": [
+                                 {
+                                    "key": "operation",
+                                    "value": {
+                                       "stringValue": "fetch"
+                                    }
+                                 },
+                                 {
+                                    "key": "type",
+                                    "value": {
+                                       "stringValue": "total"
+                                    }
+                                 }
+                              ],
+                              "startTimeUnixNano": "1661811689941624000",
+                              "timeUnixNano": "1661811689943245000"
+                           },
+                           {
+                              "asInt": "52",
+                              "attributes": [
+                                 {
+                                    "key": "operation",
+                                    "value": {
+                                       "stringValue": "query"
+                                    }
+                                 },
+                                 {
+                                    "key": "type",
+                                    "value": {
+                                       "stringValue": "total"
+                                    }
+                                 }
+                              ],
+                              "startTimeUnixNano": "1661811689941624000",
+                              "timeUnixNano": "1661811689943245000"
+                           }
+                        ]
+                     },
+                     "unit": "ms"
+                  }
+               ],
+               "scope": {
+                  "name": "otelcol/elasticsearchreceiver",
+                  "version": "latest"
+               }
+            }
+         ]
+      },
+      {
+         "resource": {
+            "attributes": [
+               {
+                  "key": "elasticsearch.index.name",
+                  "value": {
+                     "stringValue": "_all"
+                  }
+               }
+            ]
+         },
```

Comment:
We have other resource attributes already defined, some of which I suspect should be included on this resource as well. I think at least `elasticsearch.cluster.name` should be here. 

What about `elasticsearch.node.name` - is an index tied to a node?

Reply:
Update: it turns out that the name can be extracted from the same JSON as `Version` is. Do you think it's a good idea to add this additional query?

Reply:
I think that all resources collected by this scraper should always have the cluster name attribute. 

> Update: it turns out that the name can be extracted from the same JSON as Version. We could then modify the getVersion function in the scraper.go file and save it in the same way. Should it be added in this PR or in a follow-up?

This seems like a good strategy. I'm fine with adding it in this PR, since we should not emit any resources without it.